revolv/templates/base/partials/nav.html

```diff
@@ -21,7 +21,21 @@
                {% show_menu 0 100 100 100 "base/partials/menu.html" %}
            </ul>
 
-           <ul class="right">
+           <ul class="right header-small-text">
+
+               <li>
```

Comment:
Everything here except the signin/signout will be handled by django-cms, so we shouldn't hardcode it for now